server/devpi_server/main.py

```diff
@@ -462,6 +462,10 @@ def create_app(self):
         pyramid_config.add_view_deriver(self.view_deriver, under=INGRESS)
         pyramid_config.add_view_predicate('content_type', ContentTypePredicate)
 
+        # snapshoting
+        pyramid_config.add_route("/+snapshot/{user}/{index}", "/+snapshot/{user}/{index}")
```

Comment:
I would prefer the ``COPY`` HTTP method, if you have good arguments against that, then I would like ``/{user}/{index}/+copy``. Snapshotting should be an option selected with ``?snapshot``.